authui/src/authflowv2/icons/twemoji-countries.css

```diff
@@ -34,6 +34,13 @@
 }
 
 @supports (-webkit-appearance: none) and (stroke-color: transparent) {
+  @font-face {
+    font-family: "twemoji-countries";
+    src:
+      url("./icons/twemoji-countries-subset.woff2") format("woff2"),
+      url("./icons/twemoji-countries-subset.ttf") format("ttf");
```

Comment:
Then here should also be changed to `truetype`. https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/src#font_formats

Reply:
Fixed in 8c39acd